certbot/main.py

```diff
@@ -466,7 +466,7 @@ def register(config, unused_plugins):
     # We rely on an exception to interrupt this process if it didn't work.
     acc.regr = acme_client.acme.update_registration(acc.regr.update(
         body=acc.regr.body.update(contact=('mailto:' + config.email,))))
-    account_storage.save_regr(acc)
+    account_storage.save_regr(acc, acme_client)
```

Comment:
TIL `acme_client` isn't an `acme.client`. It's `certbot.client` and the `acme` client is currently `acme_client.acme` :/

This code currently blows up because `acme_client` doesn't have a directory object. We should change the name of `acme_client`, but it's up to you whether or not we do this in this PR. If you want, you can just use `acme_client.acme` here and I'll make sure this gets fixed.